things/api/http/transport.go

```diff
@@ -149,7 +150,7 @@ func MakeHandler(svc things.Service) http.Handler {
 }
 
 func decodeThingCreation(_ context.Context, r *http.Request) (interface{}, error) {
-	if r.Header.Get("Content-Type") != contentType {
```

Comment:
The same is applicable to `decodeChannelCreation` and `decodeCahnnelUpdate` methods.